app/utils/base_client2.js

```diff
@@ -79,11 +84,22 @@ const _request = function request (methodName, url, args, callback, subSegment)
   const namespace = getNamespace(clsXrayConfig.nameSpaceName)
   const clsSegment = namespace ? namespace.get(clsXrayConfig.segmentKeyName) : null
 
+  var urlOrForwardProxy
```

Comment:
same as for base_client. In fact, if it is identical, maybe make it a utility and then you could even write a test or two for it, which is probably a good idea